### ISSUE TITLE
Add possibility to write to store from non-Components

### ISSUE THREAD

**Issue by MarianBe** [Spyna/react-store#28]
I have a user-importer.js file that i call from an component and then write to store from this subcomponent. I would love to write to the store directly from my user-importer.js, which is not a component but only some functions. How would i achieve that?
The Code looks like this:
```javascript
...
async function load() {
  let userToken = await AsyncStorage.getItem('user_token')
  const parsedToken = JWTService.parseToken(userToken)
  return APIService.getOne('users', parsedToken.sub)
    .then(user => {
      return user
      // Here i want to write to store instead of returning
    })
   
}

export default {
  load
}

```

**Comment by Spyna** [comment]
Hi, I released version 1.5.0 of the lib, which gives you this possibility. 
The lib now exports the *raw* `store` and you can use it with its methods `set`, `get`, `remove` and so on. 
In any case, React must be *"in context"*.

Here's the instructions to use it: 

```JavaScript
import {store} from '@spyna/react-store'
...
async function load() {
...
store.set('the_user', user);
...
}
...
```

Please feel free to tell me if this works for you,
best
Spyna

**Comment by MarianBe** [comment]
@Spyna Very nice, works perfectly for me! Great work, thank you very much
